test/e2e/tests/test_hpo.py

```diff
@@ -143,7 +143,7 @@ def test_stopped(self, xgboost_hpojob):
         )
 
         # Delete the k8s resource.
-        _, deleted = k8s.delete_custom_resource(reference, 3, 10)
+        _, deleted = k8s.delete_custom_resource(reference, cfg.JOB_DELETE_WAIT_PERIODS, cfg.JOB_DELETE_WAIT_PERIODS)
```

Comment:
I think you meant to use cfg.JOB_DELETE_WAIT_LENGTH here
```suggestion
        _, deleted = k8s.delete_custom_resource(reference, cfg.JOB_DELETE_WAIT_PERIODS, cfg.JOB_DELETE_WAIT_LENGTHS)
```